package.json

```diff
@@ -23,13 +22,12 @@
     ]
   },
   "scripts": {
-    "prebuild": "npm run clean",
+    "prebuild": "node -e \"var s=require('shelljs'),d=['generators','es5-check','es5-test'];s.rm('-rf',d);s.mkdir('-p',d)\"",
```

Comment:
Yeah, I was honestly hoping to avoid creating new `.js` files because they'd be repeated on future projects making it harder to remove automatically in the future in projects based on the generator. The `package.json` scripts are easier to update.